### PR TITLE
Add mGetRoles and mGetProfiles

### PR DESCRIPTION
**Pull request by AnthonySendra**
* Add action `mGetProfiles` and `mGetRoles` in `securityController`
* Remove `hydrate` for roles
* Fix response from profile and role to match the new API

### CODE REVIEW

File: source/includes/_securityController.md [hunk 1]
```diff
@@ -83,6 +83,114 @@
...
+     took: 1,
+     total: 1
+  },
+  scope: null,
+  state: 'done',
+  status: 200
+}
+```
+
+Retrieve all roles with rights defined for given role `ids`.
```
Comment (scottinet) [inline source/includes/_securityController.md]: Possible better wording: `Retrieves multiple roles and their corresponding rights`

File: source/includes/_securityController.md [hunk 2]
```diff
@@ -441,17 +541,112 @@ Given a `role id`, delete the corresponding role from the database.
...
+  }
+  index: "%kuzzle",
+  collection: "profiles"
+  action: "searchProfiles",
+  controller: "security",
+  requestId: "<unique request identifier>"
+}
+```
+
+Retrieve all roles with rights defined for given role `ids`.
```
Comment (scottinet) [inline source/includes/_securityController.md]: Copy-pasted :grinning: 

Suggestion: `Retrieves multiple profiles and their corresponding roles`

Comment (benoitvidis) [inline source/includes/_securityController.md]: :+1:

**Comment by benoitvidis** [comment]
This PR is included in the Kuz 391 users crudl PR #17 one. 

Closing this one.

**Pull request closed without merging**